javascript.html.markdown

```diff
@@ -603,31 +603,39 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement without '{ }' defined will be the return statement, hence the `return` keyword is not required. 
```

Comment:
This sentence is not grammatically correct and also kind of complicated.

Alternative:
> If the body of an arrow function does not have opening and closing braces (`{}`), it can only contain a single expression. The `return` keyword cannot be used in this case. The result of the given expression will be the return value of the function.